### PR TITLE
Update to patch3rdParty messages

### PR DESCRIPTION
**Pull request by ddbb07**
- Matched naming on the Nexus with naming in the masterlist
- Added links for the source mod's
- Added forward slashes to the end of links

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -450,24 +450,24 @@ common:
...
     <<: *patch3rdParty
     subs:
       - '[Scoped Bows](https://www.nexusmods.com/skyrimspecialedition/mods/912/)'
-      - '[Scoped Bows Compatibility Kit - SBCK](https://www.nexusmods.com/skyrimspecialedition/mods/19495)'
+      - '[Scoped Bows Compatibility Kit - SBCK](https://www.nexusmods.com/skyrimspecialedition/mods/19495/)'
   - &patch3rdPartySRPC
     <<: *patch3rdParty
     subs:
-      - '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
+      - '[Spell Research](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
```
Comment (sibir-ine) [inline masterlist.yaml]: The patches on that page are specifically for version 2.2 & up of that mod, so we should specify that in the sub.
```suggestion
      - '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@ddbb07 The links to the mods aren't really necessary since the user will already have them installed & probably know where they got them from, but we can add them anyway.

**Comment by ddbb07** [comment]
> @ddbb07 The links to the mods aren't really necessary since the user will already have them installed & probably know where they got them from, but we can add them anyway.

Why i added the links to the mods was mostly for standardisation since some messages had them and others didn't but i also think it's better to have them than not.

**Comment by sibir-ine** [comment]
> Why i added the links to the mods was mostly for standardisation since some messages had them and others didn't but i also think it's better to have them than not.

@ddbb07 Yeah, it's fine. The only issue is with the changes to `&patch3rdPartySRPC`. Revert those & it'll be good to go.

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
@ddbb07 Thanks!

**Pull request merged**